### PR TITLE
feat: file path support relative path

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by atian25** [comment]
logrotator 那边也要改改

File: .autod.conf [hunk 1]
```diff
@@ -15,7 +15,6 @@ module.exports = {
     'egg-ci',
   ],
   semver: [
-    'egg-bin@1',
```
Comment (fengmk2) [inline .autod.conf]: 不支持 node 4 了？

Comment (fengmk2) [reply .autod.conf]: https://github.com/eggjs/egg-logger/blob/master/package.json#L60

看来一直不支持，应该改一下 engines

**Comment by fengmk2** [comment]
2.3.0

File: .autod.conf (hunk 1 above)
Comment (atian25) [reply .autod.conf]: 之前死马升大版本时漏了改好像

Comment (atian25) [reply .autod.conf]: https://github.com/eggjs/egg-logger/pull/42

**Pull request merged**